gradle/libs.versions.toml

```diff
@@ -57,7 +57,7 @@ jsonpath = "2.4.0"
 classgraph = "4.8.110"
 javassist = "3.28.0-GA"
 checker = "3.10.0"
-rocksdb = "7.3.1"
+rocksdb = "7.9.2"
```

Comment:
Why did you decide to update the version?

Reply:
There is a fix for the deleteRange method. It was previously implemented in a sub-optimal way that was visible with the profiler.

Reply:
More like a separate problem, not related to GC.

Reply:
Why not if we are at it?

Reply:
I think we should make a fix that is as close to the problem as possible.